frontends/ios/BitBoxApp/BitBoxApp/Bluetooth.swift

```diff
@@ -124,12 +126,25 @@ class BluetoothManager: NSObject, ObservableObject, CBCentralManagerDelegate, CB
 
     func centralManagerDidUpdateState(_ central: CBCentralManager) {
         state.bluetoothAvailable = centralManager.state == .poweredOn
+
+        if #available(iOS 13.0, *) {
+            // user denied BT permission,
+            // or restricted by device policy
+            let authorization = CBManager.authorization
+            state.bluetoothUnauthorized = (authorization == .denied || authorization == .restricted)
```

Comment:
Is this the same as `state.bluetoothUnauthorized = central.state == .unauthorized`? If not, why not?